renovate.json

```diff
@@ -1,10 +1,21 @@
 {
-  "extends": ["config:js-lib", "algolia"],
+  "extends": [
+    "config:js-lib",
+    "schedule:monthly",
+    ":semanticCommitTypeAll(chore)"
```

Comment:
does this make one grouped commit or still a commit per package?

Reply:
so that would open 50/100 PRs when the month comes around? We probably should group then right?

Reply:
No, it won't open any PR because `autoMergeType` is set to `"branch"` (see PR discription for summary of the flow). The goal is precisely to reduce noise.

Reply:
shouldn't it still be only one commit per "session" then? I feel like there will be a lot of commits (and CI runs etc) the moment in the month this happens

Reply:
We can, yes! The only reason I didn't do it is because when it fails (or succeeds but then something untested breaks), it's hard to know what dependency is to blame.

But we can try.